spec/mailers/message_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe MessageMailer, type: :mailer do
+  describe 'create' do
+    let(:message) { Message.new('Lucas', 'lucas@email.com', 'test email message') }
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.